Formula/libvirt.rb

```diff
@@ -14,6 +14,7 @@ class Libvirt < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gnutls"
+  depends_on "curl"
```

Comment:
Interesting. Can you provide the failure logs? Wondering if that's something we can fix fairly easily.